api/nodes/permissions.py

```diff
@@ -50,6 +51,24 @@ def has_object_permission(self, request, view, obj):
         return obj.is_public or obj.can_view(auth)
 
 
+class IsAdminContributor(permissions.BasePermission):
```

Comment:
This is from product - Because membership of a group can change, we are not copying the group over when a node is registered.  So because that group member is not going to be on the node, we're not allowing them to register or create draft registrations.  But there's no reason to prevent them from being viewing.

A group admin can't edit a draft, but if they're made a regular admin contrib, they can.  You can also be both a group member and a contributor, in fact, that will be common. You don't need to be removed from the group and readded as a contrib.